pkgs/build-support/agda/default.nix

```diff
@@ -70,13 +70,13 @@ let
         installPhase = if installPhase != null then installPhase else ''
           runHook preInstall
           mkdir -p $out
-          find \( ${concatMapStringsSep " -or " (p: "-name '*.${p}'") (extensions ++ extraExtensions)} \) -exec cp -p --parents -t "$out" {} +
+          find -not \( -path ${everythingFile} -or -path ${agdaLib.interfaceFile everythingFile} \) -and \( ${concatMapStringsSep " -or " (p: "-name '*.${p}'") (extensions ++ extraExtensions)} \) -exec cp -p --parents -t "$out" {} +
           runHook postInstall
         '';
       };
 in
 {
   mkDerivation = args: stdenv.mkDerivation (args // defaults args);
 
-  inherit withPackages withPackages';
+  inherit withPackages withPackages' agdaLib;
```

Comment:
Why are we returning `agdaLib` from a function that took the same `agdaLib` as an input. Doesn't seem to be much purpose for this?

Reply:
You're absolutely right, there isn't. I first defined `agdaLib` here before I copied the approach from haskell-modules. This is an artifact I forgot to undo, sorry about that.